src/UI/Wanted/Missing/MissingLayout.js

```diff
@@ -77,8 +79,16 @@ module.exports = Marionette.Layout.extend({
     },
 
     onShow : function() {
+        var self = this;
+
+        this.downloadClientSettings = new DownloadClientSettingsModel();
         this.missing.show(new LoadingView());
-        this._showToolbar();
+
+        Backbone.$.when(this.downloadClientSettings.fetch()).done(function() {
+            self.droneFolder = self.downloadClientSettings.attributes.downloadedEpisodesFolder;
+            self._showToolbar();
```

Comment:
This does add a slight delay while the settings are fetched before rendering the toolbar - feedback appreciated on if this is the best way of going about it as I wasn't too sure.

Reply:
@markus101 This one's for you, I don't know what's the best way to deal with this.

Reply:
A few ideas:

- Fetch download client settings when the app loads, much like UI settings and ensure it stays in sync when changes are made
- Fetch it when loading Wanted: Missing or settings and update it if the settings are changes. If it never changes it's cached, if it does, we can re-render and remove the button.
- Fetch during initialize then re-rendering after the request completes (if the display of the button would change)

There could be some re-rendering of the buttons if the visibility changes after the initial showing, but if we moved that button the the last position it'd just be added/removed (minus some minor styling changes)

Reply:
The caching approach is definitely the best, although I wonder if it's worth it considering v3 gets rid of the button entirely.

Two options that don't involve caching stuff:
1. Initial render with button disabled (not removed). Fetch and re-render (enabling the button) if needed.
2. Initial render with button removed. Fetch and re-render (adding the button) if needed.

1 has the least flicker, 2 is useful for most users since by now they shouldn't have the DF anymore.